tools/yarn_install.R

```diff
@@ -336,6 +337,44 @@ for (patch in list.files(patch_dir, full.names = TRUE)) {
   )
 }
 
+# ----------------------------------------------------------------------
+# Apply minification to patched files
+# ----------------------------------------------------------------------
+
+withr::with_dir("inst", {
+  local({
+    # install all deps
+    system("yarn install")
```

Comment:
Why no `--production` here if it's added above?

Reply:
So that `yarn parcel` is available (it isn't with `--production`)